website/src/components/CommunitySlide/CommunitySlide.module.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
+
+:root {
+    --primary-color: #004643;
+    --white-heading-color: #FFFFFF;
+    --off-white-color: #D9D9D9;
+    --secondary-text-color: #ABD1C6;
+    --background-color-white-theme: #ABD1C6;
+}
+
+
+
+.communityCard {
+    width: 390px;
+    height: 259px;
+    border-radius: 15px;
+    background-color: var(--primary-color);
+    margin-bottom: 4rem;
+}
+
+.ellipseContainer {
+    display: flex;
+    gap: 10px;
+    padding: 1rem;
+}
+
+.ellipse {
+    width: 10px;
+    height: 10px;
+    background-color: var(--off-white-color);
+    border-radius: 50%;
+}
+
+.borderLine {
+    width: auto;
+    border: 2px solid var(--white-heading-color);
+    background-color: var(--white-heading-color);
+}
+
+.communityCardBody {
+    margin-top: 10px;
+    margin-left: 20px;
+}
+
+.cardBodyHeader {
+    font-weight: 600;
+    font-size: 20px;
+    color: var(--white-heading-color);
+    line-height: 30px;
+    font-family: 'Poppins', sans-serif;
+    margin-top: 1rem;
+    margin-bottom: 0.3rem;
+}
+
+.cardBodyPills {
+    font-family: 'Poppins', sans-serif;
```

Comment:
Why do we want to specify `fontFamily` explicitly?

Font family of the website should be declared globally.

Reply:
Removed